share/spice/deb_version/deb_version.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_deb_version = function(api_result_str){
+        var api_result = JSON.parse(api_result_str);
+
+        if (api_result.error) {
+            return Spice.failed('deb_version');
+        }
+
+        Spice.add({
+            id: "deb_version",
+            name: "Debian Package Versions (" + api_result.package + ")",
```

Comment:
This `name` is a little too long. `Software` is more appropriate. I would move the package name to the `title`.